spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -414,6 +414,14 @@ def foo
       RUBY
     end
 
+    it 'accepts parens in splat calls' do
+      expect_no_offenses(<<-RUBY)
```

Comment:
Can we split these up into separate examples, please? 🙇

Reply:
Sure!